packages/thirdweb/src/react/web/wallets/defaultWallets.ts

```diff
@@ -19,11 +14,11 @@ export function getDefaultWallets(options?: {
   return [
     createWallet("inApp"),
     createWallet(METAMASK),
+    createWallet("io.rabby"),
```

Comment:
any reason for this? i like rainbow :D

Reply:
Mike wasn't a fan because of RainbowKit

Reply:
i think its more popular than rabby though, but lets see if ppl complain i guess

Reply:
wait but rabby above coinbase?